tools/create-api-changelog

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+import os
+import random
+
+if __name__ == "__main__":
+    dir_path = "api_docs/unmerged.d"
+    os.makedirs(dir_path, exist_ok=True)
+
+    random_hex_value = f"{random.randint(0, 0xFFFFFF):06x}"
+    file_path = f"{dir_path}/ZF-{random_hex_value}"
+
+    with open(file_path, "w") as f:
+        f.write("")
+
+    print(
+        f"""Created an empty API changelog file.
+If you've made changes to the API, document them here:
+{file_path}
+"""
+    )
```

Comment:
I think this should `git add` the new empty files -- I think that step is easy to forget, resulting in a pull request that's missing its documentation.